argilla-frontend/components/features/dataset-creation/configuration/DatasetConfigurationForm.vue

```diff
@@ -139,21 +137,21 @@ export default {
     return {
       isFocused: false,
       visibleDatasetCreationDialog: false,
-      drag: false,
     };
   },
   methods: {
     createDataset() {
       this.create(this.dataset);
     },
     addQuestion(type) {
-      const questionName = `${type} ${this.dataset.selectedSubset.questions.length}`;
+      const name = this.$t(`config.questionId.${type}`);
+      const questionName = `${name}-${this.dataset.selectedSubset.questions.length}`;
```

Comment:
I think the idea in the discussion was to use the `_` instead of `-`. Let's make the change if there are no technical limitations to that. 

```suggestion
      const questionName = `${name}_${this.dataset.selectedSubset.questions.length}`;
```